src/main/cpp/t10_unique.cpp

```diff
@@ -25,5 +25,20 @@
 using namespace std;
 
 int t10_unique() {
-
+  int qua;
+  cin >> qua;
+  int *arr = new int[qua];
+  int counter = 0;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Variable 'counter' is assigned a value that is never used.](https://app.codacy.com/manual/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4739147)